src/util.cpp

```diff
@@ -1135,41 +1591,41 @@ QString CLocale::GetCountryFlagIconsResourceReference ( const QLocale::Country e
             strReturn = "";
         }
 
-// AT LEAST QT 4.8 IS REQUIRED:
```

Comment:
Given we've moved to 5.9 minimum, I think we could delete this entire block that's commented out.  1138 to 1172 old / 1594 to 1628 new.

Reply:
not sure if we should start removing outdated comments in this PR...